tests/forte/data/multi_pack_test.py

```diff
@@ -176,6 +176,15 @@ def test_multipack_entries(self):
             ],
         )
 
+        # fix bug 559: additional test for index to pack_id changes
+        serialized_mp = self.multi_pack.to_string(drop_record=False)
+        recovered_mp = MultiPack.from_string(serialized_mp)
+        s_packs = [p.to_string() for p in self.multi_pack.packs]
+        recovered_packs = [DataPack.from_string(s) for s in s_packs]
+
+        recovered_mp.relink(recovered_packs)
```

Comment:
So what does this section of the test do? It seems that a multi pack is deserialized, but there are no assertions about the result.

Reply:
Yes the assertion code for verify is now added. Previously this is tested elsewhere however will generate exception because of #595 so the main purpose here is just not generating any exception. And now it tests deserialization also.